cpp/epidemiology/abm/household.h

```diff
@@ -0,0 +1,156 @@
+/*
+* Copyright (C) 2020-2021 German Aerospace Center (DLR-SC)
+*        & Helmholtz Centre for Infection Research (HZI)
+*
+* Authors: Daniel Abele, Sascha Korf
+*
+* Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#ifndef household_h
+#define household_h
+
+
+#endif /* household_h */
+
+#include <numeric>
+#include <vector>
+#include "epidemiology/abm/age.h"
+#include "epidemiology/utils/custom_index_array.h"
+#include "epidemiology/abm/abm.h"
```

Comment:
out convention for ordering includes: our library first, then third party, then standard.

this is so that if e.g. `age.h` uses something from the standard headers without including it, the build fails. headers should always include (or forward-declare) everything they use.